lib/modules/filteReddit.js

```diff
@@ -746,6 +755,17 @@ module.go = function() {
 			toggleNsfwFilter(toggle, true);
 		}
 	);
+
+	if (isPageType('linklist')) {
+		if (module.options.showRemoveVisitedTab.value && process.env.BUILD_TARGET !== 'safari') {
+			CreateElement.tabMenuItem({
```

Comment:
How do you feel about putting this in the floating island with the "back to top" button instead of as a tab menu item? It's been a longstanding regret to keep the "view images" button in the tab bar, especially for power-users who end up with lots of other tabs.